### ISSUE TITLE
from pymobiledevice import screenshotr will raise traceback

### ISSUE THREAD

**Issue by codeskyblue** [iOSForensics/pymobiledevice#10]
```sh
Python 2.7.11 (default, Mar 12 2016, 09:16:51) 
[GCC 4.2.1 Compatible Apple LLVM 7.0.2 (clang-700.1.81)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> from pymobiledevice import screenshotr
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/usr/local/lib/python2.7/site-packages/pymobiledevice/screenshotr.py", line 26, in <module>
    from pymobiledevice.lockdown import LockdownClient
  File "/usr/local/lib/python2.7/site-packages/pymobiledevice/lockdown.py", line 31, in <module>
    from ak_vendor.usbmux import default_mux as mux
  File "/usr/local/lib/python2.7/site-packages/ak_vendor/usbmux.py", line 279, in <module>
    default_mux = USBMux()
  File "/usr/local/lib/python2.7/site-packages/ak_vendor/usbmux.py", line 266, in __init__
    self.listener.listen()
  File "/usr/local/lib/python2.7/site-packages/ak_vendor/usbmux.py", line 224, in listen
    ret = self._exchange(self.proto.TYPE_LISTEN)
  File "/usr/local/lib/python2.7/site-packages/ak_vendor/usbmux.py", line 216, in _exchange
    self.proto.sendpacket(req, mytag, payload)
  File "/usr/local/lib/python2.7/site-packages/ak_vendor/usbmux.py", line 165, in sendpacket
    self, self.TYPE_PLIST, tag, plistlib.writePlistToBytes(payload))
AttributeError: 'module' object has no attribute 'writePlistToBytes'
```

**Comment by codeskyblue** [comment]
on python2.7 plistlib only has attribute `plistlib.plistlib.writePlistToString`

**Comment by GotoHack** [comment]
The problem comes from usbmux version bundled in the ak-vendor package. 
You must uninstall ak-vendor package : pip uninstall ak-vendor and it should work.

Fixed in commit 7ed03fde7a0943fb1e2c8ff4025134ea178d4ef2.

**Comment by codeskyblue** [comment]
thanks
